.gitmodules

```diff
@@ -12,6 +12,7 @@
 	path = ghcide
 	# url = https://github.com/alanz/ghcide.git
 	# url = https://github.com/wz1000/ghcide.git
-	url = https://github.com/haskell/ghcide.git
+	# url = https://github.com/haskell/ghcide.git
 	# url = https://github.com/fendor/ghcide.git
 	# url = https://github.com/bubba/ghcide.git
+	url = https://github.com/jneira/ghcide.git
```

Comment:
It looks like the commits are already in haskell/ghcide.

Reply:
yeah, all alternative repos could be deleted so mine too 😄